.buildkite/pipelines/fips.yml

```diff
@@ -1,32 +1,54 @@
 env:
-  DISABLE_CI_STATS_SHIPPING: "true"
+  DISABLE_CI_STATS_SHIPPING: 'true'
+  KBN_ENABLE_FIPS: 'true'
+  TEST_BROWSER_HEADLESS: 1
+agents:
+  provider: 'gcp'
+  image: 'family/kibana-fips-ubuntu-2004'
+  imageProject: 'elastic-images-prod'
 steps:
+  - command: .buildkite/scripts/lifecycle/pre_build.sh
```

Comment:
Do we also want post_build.sh?

Reply:
LGTM either way, it's an edge case.  The script itself doesn't apply for a schedule but maybe on pull requests if there's a documentation change?

Reply:
Thanks, I added it (67608c09a4bc269d64fc916ac55d19ededebc8d9) for completion sake.